ocs_ci/ocs/cluster.py

```diff
@@ -586,6 +693,230 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
+    def osd_health_check(self, count):
+        """
+        Osd health check based on pod count
```

Comment:
I'd change 'Osd' to 'OSD'. Also in other occurrences

Reply:
Done